parser.go

```diff
@@ -828,6 +851,26 @@ func (p *Parser) parseColumnOption(ctx *parseCtx, col model.TableColumn, f int)
 				l := model.NewLength(tlen)
 				l.SetDecimal(tscale)
 				col.SetLength(l)
+			} else if check(coloptValues) {
+				enumValues := make([]string, 0, 10)
+				for {
+					ctx.skipWhiteSpaces()
+					v := ctx.next()
+					if v.Type == SINGLE_QUOTE_IDENT || v.Type == DOUBLE_QUOTE_IDENT {
+						enumValues = append(enumValues, v.Value)
+					} else {
+						return newParseError(ctx, t, "expected RPAREN, SINGLE_QUOTE_IDENT, DOUBLE_QUOTE_IDENT(enum values): %s", v.Type)
+					}
+					ctx.skipWhiteSpaces()
+					commaOrRparen := ctx.next()
+					if commaOrRparen.Type == RPAREN {
+						break
+					}
+					if commaOrRparen.Type != COMMA {
+						return newParseError(ctx, t, "expected COMMA(enum values)")
+					}
+				}
+				col.SetEnumValues(enumValues)
 			} else {
 				return newParseError(ctx, t, "cannot apply coloptSize, coloptDecimalSize, coloptDecimalOptionalSize")
```

Comment:
error message should be changed to indicate that Set/Enum was considered as well.

Reply:
I agree